sdk/ai/azure-ai-client/azure/ai/client/_patch.py

```diff
@@ -185,6 +188,49 @@ def from_connection_string(cls, conn_str: str, credential: "TokenCredential", **
         project_name = parts[3]
         return cls(endpoint, subscription_id, resource_group_name, project_name, credential, **kwargs)
 
+    def upload_file(self, file_path: Union[Path, str, PathLike]) -> str:
+        """Upload a file to the Azure AI Studio project.
+           This method required *azure-ai-ml* to be installed.
```

Comment:
Can you please specify all the packages required?

Reply:
azure-ai-ml is the only package required.